docs/source/developer_guide/development-workflow.md

```diff
@@ -106,6 +106,23 @@ You can build and install all Elyra packages with:
 ```bash
 make clean install
 ```
+##### Windows Users
+
+Windows does not recognize `make` command
+
+You can install `make` on Windows with:
+
+If you have Chocolatey:
+
+```bash
+choco install make
+```
+
+or 
+
+```bash
+conda install m2-base
```

Comment:
'm2-base' by itself does not contain 'make', you need to also install 'm2-make' to enable the make command in windows. I would suggest also adding '-y' on the command to avoid confirmation prompt.

Reply:
Could this be a case for `npm scripts` over `make`? I guess this isn't a huge hurdle, but just one less dependency to install for windows users